utils/pytorch.py

```diff
@@ -43,8 +43,10 @@ def __init__(self, model, disable_jit_freeze=False, example_inputs=None, func=No
                 print(f"Loaded from cached file at {cached_path}")
             else:
                 try:
+                    if skip_script:
```

Comment:
A bit convoluted way of doing this. With both disable_jit_freeze and skip_script args now Im thinking that perhaps freezing facilities should be taken out of __init__ and instead be handled by dedicated function of PyTorchRunner class on demand. Ie. when someone wants to actually freeze the net they should init pytorchRunner, and then call runner.freeze("tracing") runner.freeze("script").

Reply:
Or smth similar.